dali/daliadmin/daadmin.cpp

```diff
@@ -258,7 +258,7 @@ void _delete_(const char *path,bool backup)
 
 //=============================================================================
 
-void set(const char *path,const char *val)
+void setValue(const char *path,const char *val,StringBuffer &out)
```

Comment:
minor: effectively returning old values, normally if it's returning something, we'd pass in StringBuffer reference, but also return it, i.e. StringBuffer &setValue( ..